packages/flutter_markdown/lib/src/widget.dart

```diff
@@ -46,7 +46,11 @@ typedef MarkdownCheckboxBuilder = Widget Function(bool value);
 /// Signature for custom bullet widget.
 ///
 /// Used by [MarkdownWidget.bulletBuilder]
-typedef MarkdownBulletBuilder = Widget Function(int index, BulletStyle style);
+typedef MarkdownBulletBuilder = Widget Function(
+  int index,
+  BulletStyle style,
+  int nestLevel,
```

Comment:
I haven't done a full review here, but I assume the breaking change is that we're adding a parameter to a method that package clients need to implement?

If so, it would be better to make the breaking change replace it with a parameter object, so that next time it doesn't need a breaking change.

Reply:
I agree.
Which is better to use class or record, as a parameter object here?

Reply:
Adding a field to a record later would be breaking (since it changes the type), so it needs to be a class to get the benefit.